src/utils/CMakeLists.txt

```diff
@@ -15,3 +15,16 @@ target_link_libraries (
   )
 smf_seastar_target(smf_utils)
 install(TARGETS smf_utils DESTINATION lib)
+set(smf_utils_headers
+  ${PROJECT_SOURCE_DIR}/src/utils/caching/clock_pro/chunk.h
```

Comment:
I'm assuming you want this directory structure retained under the installation path?

Reply:
Yes. Good thinking. Will fix.